### PR TITLE
[#6604] add captcha to forms

### PR DESCRIPTION
**Pull request by goapunk**
- [x] add captcha to forms
- [ ] add captcha to newsletter block?

### CODE REVIEW

File: apps/forms/models.py [hunk 1]
```diff
@@ -21,8 +23,23 @@ class FormField(AbstractFormField):
...
 
+class WagtailCaptchaFormBuilder(FormBuilder):
+    @property
+    def formfields(self):
+        # Add captcha to formfields property
+        fields = super().formfields
+        fields['captcha'] = CaptcheckCaptchaField(
+            label=_('I am not a robot'),
+            help_text=_('If you are having difficulty please contact us, '
+                        'details adjacent'))
```
Comment (fuzzylogic2000) [inline apps/forms/models.py:34]: I think the details adjacent is true for a+, but not here?

**Review by fuzzylogic2000 — COMMENTED** [review]
I did rebase, and it is mainly working! But the captcha is not loaded. Did I mess that up with the rebase?

File: apps/forms/models.py (hunk 1 above)
Comment (goapunk) [reply apps/forms/models.py:34]: yes, and we also need to remove that anyway as it doesn't pass the bitv test

**Comment by goapunk** [comment]
> I did rebase, and it is mainly working! But the captcha is not loaded. Did I mess that up with the rebase?

it seems I didn't add it anywhere yet, or maybe forgot to commit it :man_shrugging:

**Comment by goapunk** [comment]
ok, I forgot to include the tag which loads the js, with it it works

File: apps/forms/models.py (hunk 1 above)
Comment (goapunk) [reply apps/forms/models.py:34]: todo: need wording here

**Comment by goapunk** [comment]
ok, this should be ready for another review

File: apps/forms/templates/apps_forms/includes/form_field.html [hunk 2]
```diff
@@ -33,6 +33,36 @@
     {% endif %}
 </fieldset>
 <hr>
+{% elif field|is_captcha %}
+<div class="form-group">
```
Comment (goapunk) [inline apps/forms/templates/apps_forms/includes/form_field.html:37]: I copied this whole block from a+, so not super sure if we need all of this

**Review by phillimorland — COMMENTED** [review]
it uses `Ubuntu,Roboto,Arial,sans-serif;` that font stack, so we should probably change that. Also confirmation page causes error but dont think thats to do with captcha
`TypeError at /en/contact-form/
FormSubmission() got an unexpected keyword argument 'email'`

**Comment by goapunk** [comment]
I fixed the errors and removed the font-family

**Review by phillimorland — APPROVED** [review]
looking good, it's in english on the German site, once that's sorted good to go

**Comment by fuzzylogic2000** [comment]
@phillimorland @goapunk This is good to go as the translations are done in weblate now, right?

**Comment by phillimorland** [comment]
@fuzzylogic2000 we don't have js translations on DS so not sure?!

**Comment by goapunk** [comment]
@fuzzylogic2000 @phillimorland will add the js translations in weblate

**Pull request merged**